src/WCSimPhysicsList.cc

```diff
@@ -66,7 +66,7 @@ void WCSimPhysicsList::ConstructProcess()
 {
   //AddTransportation();
   ConstructEM();
-  ConstructlArStepLimiter();
+  // ConstructlArStepLimiter();
```

Comment:
That was an artefact when I was debugging the new particleIterator object. This method works fine and should be uncommented.